openfisca_core/indexed_enums/enum_array.py

```diff
@@ -117,6 +122,12 @@ def __eq__(self, other: object) -> bool:
             >>> array = numpy.array([1])
             >>> enum_array = enum.EnumArray(array, Housing)
 
+            >>> enum_array == Housing
```

Comment:
I do not understand what this comparison purpose is ?

Reply:
It is just to illustrate that comparison will always behave vectorially. I spend quite some time tracking a bug related to automating castings, and realised that `numpy` has been changing behaviour towards always give back an array. I can delete the example, however the feature comes with subclassing `numpy.ndarray`.